src/NuGet/Microsoft.CodeAnalysis.Workspaces.Common.nuspec

```diff
@@ -27,12 +27,8 @@ Supported Platforms:
     <serviceable>true</serviceable>
   </metadata>
   <files>
-    <file src="Microsoft.CodeAnalysis.Workspaces.dll" target="lib\portable-net45+win8" />
-    <file src="Microsoft.CodeAnalysis.Workspaces.xml" target="lib\portable-net45+win8" />
-    <file src="Microsoft.CodeAnalysis.Workspaces.dll" target="lib\net45" />
-    <file src="Microsoft.CodeAnalysis.Workspaces.xml" target="lib\net45" />
-    <file src="Microsoft.CodeAnalysis.Workspaces.Desktop.dll" target="lib\net45" />
-    <file src="Microsoft.CodeAnalysis.Workspaces.Desktop.xml" target="lib\net45" />
+    <file src="Microsoft.CodeAnalysis.Workspaces.Desktop.dll" target="lib\net46" />
```

Comment:
There should also be a copy of Microsoft.CodeAnalysis.Workspaces.dll in the lib\net46 target as well. The .Desktop API is additive, not a different implementation of the same. #12444 is tracking a further cleanup, but you'll want to do that minimally.